Algo-Workout/AW-T3
# Added bulletin board form

##  Add Bulleting board Create Post feature


**Description**
- Moved Form to Dashboard page, viewable upon login
- Created route to get all posts from the database with the most recent options first
- Created route to add a post to the database
- Displayed all posts on the front end

**Steps to Validate Feature**
1. ```npm install```, ```npm run dev```
2. Form should be viewable under the bulleting board. Enter a short sentence or two and press submit.
3. Then the bulletin board display should refresh with the new submission

<img width="1080" alt="image" src="https://github.com/Algo-Workout/AW-T3/assets/2013689/3c512227-2061-45e3-b5fb-538b599f7f59">

4. Use ```npx prisma studio``` to view delete and edit all current posts in data base

The other unfinished version with the modal is in the Dwayne/ModalWindow branch
